apps/nameguard.io/src/app/components/Hero.tsx

```diff
@@ -20,31 +22,18 @@ export function Hero() {
             Guard your users from heartbreak and keep ENS usage safe across web3
           </p>
           <HeroStartCommand />
-          <a
-            href="https://api.nameguard.io/docs"
-            className="hidden search_bar_change:block relative z-10"
-          >
-            <button className="flex justify-center items-center px-[25px] py-[13px] rounded-lg bg-black z-10 shadow-sm transition hover:bg-gray-800 cursor-pointer">
-              <p className="text-white not-italic font-medium text-base leading-6">
-                View the docs
+          <div className="max-w-md">
+            <div className="flex flex-col items-center gap-3 self-stretch">
+              <div className="search_bar_change:hidden">
+                <NGSearch />
+              </div>
+              <p className="w-full h-fit text-gray-500 text-sm leading-6 font-normal text-center gt_mobile:font-light search_bar_change:hidden">
+                or
               </p>
-            </button>
-          </a>
-          <div className="flex search_bar_change:hidden flex-col items-center gap-3 self-stretch">
-            <NGSearch />
-            <p className="w-full h-fit text-gray-500 text-sm leading-6 font-normal text-center gt_mobile:font-light">
-              or
-            </p>
-            <a
-              href="https://api.nameguard.io/docs"
-              className="relative z-10 w-full h-fit max-w-xs"
-            >
-              <button className="w-full h-fit box-border flex justify-center items-center self-stretch px-[17px] py-[9px] rounded-lg bg-black z-10 shadow-sm transition hover:bg-gray-800 cursor-pointer">
-                <p className="text-white not-italic font-medium text-base leading-6">
-                  View the docs
-                </p>
-              </button>
-            </a>
+              <NextLink href="https://api.nameguard.io/docs">
```

Comment:
Maybe this is a stupid question, but my understanding is that this would be a button rather than a "NextLink"? In my mental model a "NextLink" would be for a "text link", for example such as the one here:

![CleanShot 2024-05-21 at 19 30 48](https://github.com/namehash/nameguard/assets/126201998/0215ddfe-d85e-45c2-b0ef-aaee21243844)

Perhaps I'm thinking about this wrong. Appreciate your advice :+1: